data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/damage/damage_wfl_self.cfg

```diff
@@ -0,0 +1,31 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [damage]value=
+##
+# Actions:
+# Give the leaders a damage ability that does a quarter its own max hp.
+# Have the side 1 leader attack the side 2 leader with both its weapons
+##
+# Expected end state:
+# The side 1 leader has 2 weapons both of which have 250 damage (1000/4), self is the attacker
+#####
+{COMMON_KEEP_A_B_UNIT_TEST "damage_wfl_self" (
+    [event]
+        name = start
+
+        [modify_unit]
+            [filter]
+            [/filter]
+            [effect]
+                apply_to = new_ability
+                [abilities]
+                    {TEST_ABILITY damage "(max_hitpoints/4)" () SELF=yes}
```

Comment:
if both units have same max_hitpoints this does not test which unit is used

Reply:
Updated.